pages/login.js

```diff
@@ -0,0 +1,30 @@
+import { useSession, signIn, signOut } from "next-auth/react";
+import Button from "@/components/Button";
+import { useRouter } from "next/router";
+export default function Login() {
+  const { data: session } = useSession();
+  const router = useRouter();
+
+  return (
+    <>
+      {session ? (
+        <>
+          <p>Signed in as {session.user.name} </p> <br />
+          <button onClick={() => signOut()}>Sign out</button>
+        </>
+      ) : (
+        <>
+          <p>Not signed in </p>
+          <br />
```

Comment:
<br> is not a element you should use for layout. 
please - when you are styling your all - get rid of all the `br` and get real stying done ;)